scripts/gen_circleci_config.py

```diff
@@ -79,6 +79,11 @@ def check(name: str, command: str, paths: t.Set[str]) -> None:
         command="riot -v run -s scripts",
         paths={"scripts/*.py"},
     )
+    check(
+        name="Run conftest tests",
+        command="riot -v run -s conftest",
+        paths={"tests/conftest.py", "tests/meta/*"},
```

Comment:
```suggestion
        paths={"tests/*conftest.py", "tests/meta/*"},
```
should future-proof this?